presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+
+* Fix incorrect results when dividing certain decimal numbers (:pr:`12930`).
+* Fix planning failure for certain join queries caused by invalid distribution
+  type (:pr:`12354`).
+* Add support for automatic retries for failed buckets when using grouped
```

Comment:
A few comments
1. The first sentence is a bit hard to parse
2. Is there anything more specific to describe what's supported other than "certain queries"?
3. We usually start with the configuration properties and then note the functionality https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#phrasing (though the description there seems like its saying the opposite of what the example does, the example is what I've seen us do in other release notes).  I don't personally care either way, but we should be consistent.

Something like this 
```
Add  session property ``recoverable_grouped_execution`` [and configuration property...] to enable automatically retrying failed buckets when using grouped execution.  Currently this is only supported in the Hive connector and for queries that <whatever characterizes those "certain queries"> (:issue:`12124`).
 ```

Reply:
we concluded that if the note is about a new feature and not mostly about configuring the cluster, then the description could go first.  We'll update the release note guidelines to be clearer.